index.html

```diff
@@ -1342,10 +1342,11 @@ <h4>skolemizeExpandedJsonLd</h4>
 
           <p>
 The following algorithm replaces all blank node identifiers in an expanded
-JSON-LD document with custom-scheme URNs. The required inputs are an expanded
-JSON-LD document (<var>expanded</var>), a custom URN scheme
-(<var>urnScheme</var>), a UUID string or other comparably random string
-(<var>random</var>), and reference to a shared integer (<var>count</var>).
+JSON-LD document with custom-scheme URNs and assigns custom-scheme identifiers to
+blank nodes. The required inputs are an expanded JSON-LD document
+(<var>expanded</var>), a custom blank node prefix
+(<var>bnPrefix</var>), a UUID string or other comparably random string
```

Comment:
So the reason for saying "custom URN scheme" in the current text is to make it clear that the output is a URI (specifically, a URN here) and to do it in a consistent fashion. Skolemization transforms a blank node label to a URI, which is the important bit of this operation as ensures subsequent JSON-LD processing will not change the label.

Changing the text to "custom blank node prefix" here and the param name to `bnPrefix` implies that the blank node labels could just changing from one blank node label to another, when they are vitally changing to URIs. It's true that the text above would still say that this algorithm will transform them to "custom-scheme URNs" -- but it seems a little less clear.

What's the reason for changing from `urnScheme` to `bnPrefix`? Is it to try and simplify the text? If so, alternatively, we could say `urnPrefix` (and clearly state that it must start with `urn:`). This would make clear that the function has the same behavior as before and it wouldn't introduce the possibility for it to be used in a potentially erroneous way. Thoughts?

Reply:
Hi @dlongley thanks for the explanations. When I first encountered this function my first questions what "where do the blank node ids get assigned?" and it turns out that this function will do that if there are not. So wanted to highlight this important piece of information. 

`urnScheme` versus `bnPrefix`: DB's example code was taking a `prefix` as an argument so with that and the discrepancy between where to put the underscores, "_", I thought it might be better to be more like the example code parameters. So for my implementation I made a `bnPrefix` parameter.  As for proper terminology and naming I'll go with whatever you prefer.

Reply:
@Wind4Greg,

I think we should keep the existing `urnScheme` approach and just have the underscore bug fix and `skolemizedExpandedDocument` naming fix here.

Reply:
Hi @dlongley I put back the previous text and `urnScheme` input parameter (left in the other fixes). However, would still like some clarifying text that says this function actually assigns blank node ids to blank nodes (in appropriate wording). Otherwise it seems like there is some other function that assigns blank node ids. Can you suggest some text about this with the appropriate wording? Cheers Greg